experiments.json

```diff
@@ -238,5 +238,20 @@
       "threshold": 0.2,
       "description": "Edit Top Sites is enabled"
     }
+  },
+  "pocketStories": {
+    "name": "Trending stories provided by Pocket",
+    "active": true,
+    "description": "Display recommendations of trending stories by Pocket",
+    "control": {
+      "value": true,
+      "description": "Pocket trending stories active"
+    },
+    "variant": {
+      "id": "exp-017-pocketstories",
+      "value": true,
+      "threshold": 0,
```

Comment:
Okay then yes let's leave it as it is for now.  We'll need to find a better way to be able to separate being able to manually flag something on vs setting it as an active experiment, but we can deal with that later.